Cron/UpdateOrdersToProcessing.php

```diff
@@ -143,7 +147,15 @@ public function execute()
                 {
                     $rzpWebhookDataObj = unserialize($rzpWebhookData);
 
-                    $this->updateOrderStatus($order, $rzpWebhookDataObj);
+                    if (isset($rzpWebhookDataObj['payment.authorized']) === true)
+                    {
+                        $this->updateOrderStatus($order, 'payment.authorized', $rzpWebhookDataObj['payment.authorized']);
+                    }
+                    
+                    if (isset($rzpWebhookDataObj['order.paid']) === true)
+                    {
+                        $this->updateOrderStatus($order, 'order.paid', $rzpWebhookDataObj['order.paid']);
```

Comment:
@nikhilsharma1100 Please include value for `event` using constant variable
Example: `static::ORDER_PAID`

Reply:
done